config/phenomenal_tools2container.yaml

```diff
@@ -33,6 +33,14 @@ assignment:
   max_pod_retrials: 3
   
 # MTBLS/ISA
+- tools_id:
```

Comment:
This entry is duplicated in line 74 below. Please remove the one you don't want to use (hopefully remove the one with latest and leave a the one with a version tag).

Reply:
Done, I've kept the one with the version tag.